### ISSUE TITLE
[FEATURE] Change handling of unofficial IWAD files

### ISSUE THREAD

**Issue by Mastrius0713** [odamex/odamex#464]
Testing a custom IWAD using the -IWAD command line option and it loaded DOOM2.WAD as the IWAD while loading the custom IWAD like a PWAD. Does not happen with official IWADs or FreeDoom IWADs.

**Comment by Ch0wW** [comment]
This isn't a bug.

When using `-iwad`, it checks a list of Hardcoded IWADs we support; hence the fact it worked on Freedoom or the official ones without issues.

EDIT: what were you trying to launch, out of curiosity?

**Comment by leximayfield** [comment]
Yeah the fundamental problem is that when someone provides a custom IWAD, Odamex has no way to tell what kind of IWAD-specific settings to use.  So either there should be a reasonable default, or we need some mechanism to detect what settings to use with an alien IWAD.

**Comment by rakohus** [comment]
Maybe the solution is for it to read from a generic mapinfo lump that is similar to Doom2's and then leave it up to the user to supplement their own new mapinfo lump. We already include these types of lumps: `_D1NFO, _D2NFO, _PLTNFO` etc

**Comment by rakohus** [comment]
Also changing this to a feature request rather than a bug.
